handlers/parse/guest_link.go

```diff
@@ -18,3 +23,20 @@ func GuestLinkLabel(label string) (picoshare.GuestLinkLabel, error) {
 
 	return picoshare.GuestLinkLabel(label), nil
 }
+
+func GuestFileLifeTime(fileLifetimeRaw string) (picoshare.FileLifetime, error) {
```

Comment:
Can we rename this to `FileLifetime`? There's nothing specific about guest links to this function. It could even live in a different file.

Reply:
Renamed to `FileLifetimeFromString` as `FileLifetime` already exists. Moved to `file_lifetime` file.